### PR TITLE
Add rcon command function with layer

### PR DESCRIPTION
**Pull request by brandonsturgeon**

### CODE REVIEW

File: cfc-nanny/layers/lambda_responses/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .lambda_responses import *
```
Comment (plally) [inline cfc-nanny/layers/lambda_responses/__init__.py:1]: i dont think you need this init file, or the init files in other layers

File: cfc-nanny/layers/response_caching/response_caching.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import time
+
+class ResponseCache():
```
Comment (plally) [inline cfc-nanny/layers/response_caching/response_caching.py:3]:
```suggestion
class ResponseCache:
```

File: cfc-nanny/template.yaml [hunk 3]
```diff
@@ -18,17 +18,82 @@ Parameters:
...
+    Default: 'CFCTestRconPassword'
+
 Globals:
   Function:
     Timeout: 60
     Runtime: python3.8
 
-_shared:
-  restart: &restart_defaults
+Metadata:
```
Comment (plally) [inline cfc-nanny/template.yaml:34]: will this just not work at all if its not under Metadata, feels weird putting stuff under a key thats used for other things

File: cfc-nanny/template.yaml [hunk 4]
```diff
@@ -18,17 +18,82 @@ Parameters:
...
+      - !Ref RconDepLayer
+      - !Ref LambdaResponseLayer
+
+  rcon_variables:
+    cfc3: &cfc3_rcon
+      RCON_PORT: "27015"
+      RCON_IP: "172.245.14.234"
+      RCON_PASSWORD: !Ref CFC3RconPassword
+
+    cfctest: &cfctest_rcon
```
Comment (plally) [inline cfc-nanny/template.yaml:68]: might be clearer if this was called cftest_rcon_env and cfc3_ron_env 
because CFCTestRcon is a function

**Review by plally — CHANGES_REQUESTED** [review]

File: cfc-nanny/template.yaml (hunk 3 above)
Comment (brandonsturgeon) [reply cfc-nanny/template.yaml:34]: Yeah it's super strict about which keys are allowed, so anything other than the allowed keys will throw an error during deployment. This is the only place that it works

File: cfc-nanny/layers/lambda_responses/lambda_responses.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
+from json import dumps
+
+class ResponseBuilder:
+    def build(self):
+        body = {}
+        content = self.content
+        errors = self.errors
```
[the remark this answers is not in the document]
Comment (brandonsturgeon) [reply cfc-nanny/layers/lambda_responses/lambda_responses.py:7]: This whole thing is a complicated way of avoiding just having a function, so it's already a little wonky.

In this case, though, I think duplicating the init methods doesn't add any value. `Response` already takes those values in and defaults them to `None`.

**Pull request merged**